src/datastore_mad/remotes/lxd_downloader.sh

```diff
@@ -155,7 +155,7 @@ rm -f /etc/resolv.conf > /dev/null 2>&1
 echo "nameserver $DNS_SERVER" > /etc/resolv.conf
 
 apt-get update > /dev/null
-apt-get install curl -y > /dev/null 2>&1
+apt-get install curl openssh-server -y > /dev/null 2>&1
```

Comment:
At the begning of the script there is a configuration section, please add there the extra packages needed:

```
#-------------------------------------------------------------------------------
# Downloader configuration attributes
#-------------------------------------------------------------------------------
...
PACKAGES_DEBIAN_UBUNTU="curl openssh-server"
PACKAGES_CENTOS6="epel-release openssh-server"
```